ndarray-rand/README.rst

```diff
@@ -18,6 +18,7 @@ Recent Changes
 - 0.10.0
 
   - Require rand 0.7
+  - Require Rust 1.32 or later
```

Comment:
Should we put this in the `rand` changelog too?

Reply:
I'm not sure what you mean. This is in the `ndarray-rand` changelog.

Reply:
sry, I had meant the `ndarray` changelog

Reply:
Oh, yes, when we write up the changes for` ndarray` 0.13, we should include the Rust version requirement change.

Reply:
OK great. Given that's moving in another PR, we can wait until that's merged to add?

Reply:
Yes, we can include it as part of another PR updating the `ndarray` changelog.

Reply:
Cool. Anything left from this PR?